test/integration/krane_deploy_test.rb

```diff
@@ -1467,19 +1448,6 @@ def test_validation_failure_on_sensitive_resources_does_not_print_template
     refute_logs_match("YWRtaW4=")
   end
 
-  def test_render_failure_on_sensitive_resource_does_not_print_template
```

Comment:
Is this particular case (render failure not showing secrets) being handled in any other test?

Reply:
I think in this case, I forgot to recreate this in the render tests 😬 

👀

Reply:
@dirceu so the reason I removed this is because the test on line 1439 tests that validation failure on a sensitive resource doesn't print the template.
This test tests that a render failure on a sensitive resource doesn't print the template

This isn't tested anywhere else so I'm going to add this back and get this minimally moved over to the render task test in a different PR.

Make sense?

Reply:
Yes it does, thank you!